ci-skip github action marking builds as failed

The current `skip ci` github action marks the commit as `failed` when it skips a build -- we should find a better way to handle that so it doesn't mark commits as bad

I think this is the problem : 
[skip ci] action fails with code 78. See the error ["##[error]Docker run failed with exit code 78"] in your commit failures. This used to be recognized as a neutral code and recently Github stopped treating it as that. See this : https://twitter.com/ethomson/status/1163899559279497217 . They will come up with a different strategy to detect neutral exits like [skip ci].
 
One work around is to skip-checks for that commit since you don't want to run ci anyway. See this : https://help.github.com/en/articles/about-status-checks#skipping-and-requesting-checks-for-individual-commits . Not very sure but this might help as a temporary fix.

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -2,11 +2,14 @@ name: Docs
 
 on:
   push:
+    paths:
+      - packages/docs/**
     branches:
       - master
 
 jobs:
   default:
+    if: "!contains(github.event.commits[0].message, '[skip ci]')"
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1

--- a/.github/workflows/push.yml
+++ b/.github/workflows/push.yml
@@ -4,6 +4,7 @@ on: [push]
 
 jobs:
   default:
+    if: "!contains(github.event.commits[0].message, '[skip ci]')"
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1

